selfconference/selfconf-ios
# Update layouts for iPhone X/iOS 12.2

## What does this PR do :question:
This PR updates the views and layouts for the full screen experience on the iPhone X, as well as view settings that are compatible with iOS 12.2. Changes include:

- Replacing Top and Bottom Layer Guides with Safe Area Guides
- Adding an asset catalog color sets
- Bound views into safe areas
- Extending view background colors beyond the safe area
- Small tweaks to modernize look (ex: removing translucence from 'Code of Conduct' navigation bar)


## How to test it :microscope:
1. Run the app in the iPhone X simulator
2. Note that the full screen has a teal background color (see gif)
3. Tap the different segments of the segmented control. Note that a collection of cards should appear for every given day (see gif)
4. Tap any card and tap the empty heart icon to add the event to 'Favorites', then tap 'Favorites'. Note that the same event card appears (see gif)
5. Tap any card and tap 'Submit Feedback' near the bottom of the card. Note that a 'Submit Feedback' view appears (see gif)
6. Tap the 'Code of Conduct' icon, and note that the navigation bar appears beneath the safe area (see gif). Exit the 'Code of Conduct view by tapping the 'X' icon
7. Tap the search bar and search for any speaker name or keyword from any event card. Note that a valid result appears (see gif)


## Any background info you would like to include :white_check_mark:
This PR contains changes related to the views only. No functionality was added or changed!

The issue for this task is here: https://github.com/selfconference/selfconf-ios/issues/6


## Screenshots (if applicable) :camera:
![selfconf-pr-update-views](https://user-images.githubusercontent.com/8409475/57973602-6c826780-7979-11e9-9ee1-e0fd6409bf17.gif)
